tests/loader/test_1dsfm_loader.py

```diff
@@ -0,0 +1,78 @@
+"""Unit tests for the OneDSFM Loader class.
+
+Authors: Yanwei Du
+"""
+
+import unittest
+from pathlib import Path
+
+from gtsfm.loader.one_d_sfm_loader import OneDSFMLoader
+
+"""The unit test data contains four images and three of them have valid exif data """
+
+DATA_ROOT_PATH = Path(__file__).resolve().parent.parent / "data"
+
+DEFAULT_FOLDER = DATA_ROOT_PATH / "1dsfm"
+
+class TestFolderLoader(unittest.TestCase):
```

Comment:
Can we rename this as TestOneDSfMFolderLoader?

Reply:
Done